src/libmdb/backend.c

```diff
@@ -727,6 +760,28 @@ mdb_get_relationships(MdbHandle *mdb, const gchar *dbnamespace, const char* tabl
 
 			break;
 		}
+
+		/* On some databases (eg PostgreSQL) we also want to set
+		 * the constraints to be optionally deferrable, to
+		 * facilitate out of order bulk loading.
+		 */
+		switch (backend) {
+			case MDB_BACKEND_POSTGRES:
+				{
+					gchar *add_constraint;
+					add_constraint = text;
+					text = g_strconcat(add_constraint,
+						"ALTER TABLE ", quoted_table_1,
+						" ALTER CONSTRAINT ",
+						quoted_constraint_name,
+						" DEFERRABLE"
+						" INITIALLY IMMEDIATE;\n",
+						NULL);
+					g_free(add_constraint);
+				}
```

Comment:
Needs a `break;` statement here

Reply:
See [comment on pull request that I've just made](https://github.com/mdbtools/mdbtools/pull/93#issuecomment-749213676).   If you want to fix it up, feel free.  Otherwise, close the 4.5 year old ticket.  I'm going to unwatch this pull request now.

Ewen